recipes/libtiff/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "4.0.6" %}
+
+package:
+    name: libtiff
+    version: {{ version }}
+
+source:
+    fn: tiff-{{ version }}.tar.gz
+    url: ftp://ftp.remotesensing.org/pub/libtiff/tiff-{{ version }}.tar.gz
+    md5: d1d2e940dea0b5ad435f21f03d96dd72
+    patches:
+        - def_snprintf.patch  # [win]
+
+build:
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python  # [win]
+        - cmake  # [win]
+        - zlib
+        - jpeg
+        - xz  # [not win]
+    run:
+        - zlib
+        - jpeg
+        - xz  # [not win]
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/libtiff.a  # [unix]
+        - test -f ${PREFIX}/lib/libtiffxx.a  # [unix]
+        - test -f ${PREFIX}/lib/libtiff.so  # [linux]
+        - test -f ${PREFIX}/lib/libtiff.dylib  # [osx]
+        - test -f ${PREFIX}/lib/libtiffxx.so  # [linux]
+        - test -f ${PREFIX}/lib/libtiffxx.dylib  # [osx]
+        - conda inspect linkages -n _test libtiff  # [linux]
+
+about:
+    home: http://www.remotesensing.org/libtiff/
+    license: HPND
+    summary: Support for the Tag Image File Format (TIFF)
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
```

Comment:
@stuarteberg since you created this recipe can I add you as a maintainer :wink:?